pinot-core/src/main/java/org/apache/pinot/core/plan/TimeSeriesPlanNode.java

```diff
@@ -66,7 +66,8 @@ _segmentContext, _queryContext, getProjectPlanNodeExpressions(), DocIdSetPlanNod
         getGroupByColumns(),
         _timeSeriesContext.getTimeBuckets(),
         projectionOperator,
-        _seriesBuilderFactory);
+        _seriesBuilderFactory,
+        _segmentContext.getIndexSegment().getSegmentMetadata());
```

Comment:
indexes on the column will work as it works in V1 or MSE for time series engine as well correct? 
Can you share what is the impact of this change?

Reply:
No impact.. this is just passing some metadata